File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.cpp
#include <bits/stdc++.h>

using namespace std;

bool solution(string s) {
    for(int i = 0; i < s.size(); i++){
        if(s[i] > '9'){
            return false;
        }
    }
    if(s.size() != 4 && s.size() != 6)
        return false;
    return true;
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        boolean answer = true;
        char[] arr = s.toCharArray();
        for(int i = 0; i < arr.length; i++){
            if(arr[i] > '9')
                return false;
        }
        if(arr.length != 4 && arr.length != 6)
            return false;
        return answer;
    }
}